godel_scan_analysis/include/godel_scan_analysis/scan_utilities.h

```diff
@@ -0,0 +1,52 @@
+#ifndef SCAN_UTILITIES_H
+#define SCAN_UTILITIES_H
+
+#include <vector>
+
+namespace rms
+{
+
+template<typename FloatType>
+struct Point
```

Comment:
It didn't seem worth adding a dependency to something else just for something as simple as a pair (x,y). Furthermore, I'll probably strip it out at some point and just have continuous buffers of x and y.